.idea/runConfigurations/app_emulator.xml

```diff
@@ -62,7 +62,7 @@
     <option name="SEARCH_ACTIVITY_IN_GLOBAL_SCOPE" value="false" />
     <option name="SKIP_ACTIVITY_VALIDATION" value="false" />
     <method v="2">
-      <option name="RunConfigurationTask" enabled="false" run_configuration_name="seedvault [installEmulatorRelease]" run_configuration_type="GradleRunConfiguration" />
+      <option name="Gradle.BeforeRunTask" enabled="false" tasks="installEmulatorRelease" externalProjectPath="$PROJECT_DIR$/app" vmOptions="" scriptParameters="" />
```

Comment:
Why is this needed? It wouldn't install the app otherwise?

Reply:
I'd have to check again but I recall using regular `adb install` wasn't sufficient as it would remove it from `priv-app` (I think... I didn't investigate much past it not working).

Reply:
I actually sometimes/often have the issue that Android studio says it installed just fine, but the app wasn't really updated. Maybe this fixes it?

Reply:
Could be yeah! I haven't tested on a real device since I only have one, but it's worth trying!